cds-snc/cppd-infrastructure
# Policy name fix

## Description 

Added some tests to enforce ESDC naming conventions and auto-tagging policies. These will ensure people are happy with us and also will ensure the auto-tagging doesn't blow away our stuff.

Wrote a bunch of policys to validate the following:

- Names start with ES or es
- 3rd char in a name is S for items tagged Sandbox
- 4th char in a name is C for canadacentral and E for canadaeast

Fixed names so the prefixes are all templated and driven from varables

Add prefixes for other tags
Fix to match offical tagging policy when it get's implemented

# Test Instructions

1. Change some of the values in terraform.tfvars so they are wrong. 
1. like have an environment of Sandbox but a environmentprefix of D and see if it fails
